deploy/stop

```diff
@@ -3,6 +3,10 @@ set -euo pipefail
 
 . "$(dirname "${0}")/utils.sh"
 
+if has_namespace $APP_NAMESPACE_NAME; then
```

Comment:
Please add `[  ]` for the sake of uniformity in the codebase

Reply:
This is the way we are using the function` has_namespace` in our code 
You don't use [[ (or [) when running a command and checking the result code.